week3/homework/src/actions.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const { readFile, writeFile } = require('fs');
+const { promisify } = require('util');
+const uuid = require('uuid/v4');
+const read = promisify(readFile);
+const write = promisify(writeFile);
+const FILE_PATH = 'todo.json';
+const list = {};
+
+
+const fileContent = (todo) => {
```

Comment:
`fileContent` is not the best name for a function. Function names should be actions, e.g. `writeToDo`.

Reply:
Changed this too.